src/ReactToPdf.jsx

```diff
@@ -57,6 +71,8 @@ ReactToPdf.defaultProps = {
   options: undefined,
   x: 0,
   y: 0,
+  imgWidth: 300,
```

Comment:
I appreciate the effort here, but we should support multi-page feature without having to specify any dimensions on the image dimensions and specially using props. I'd suggest taking a looking at jsPdf internal page values together with the canvas dimensions to create a more generic usage of this component.

Reply:
Took 5 months to get review on this, I ended up doing a fork for my needs on a project.

This PR was a clone of another issue we let go stale. 

Maybe done is better than perfect? Could you open an issue for making it generic as a fast follow?

Reply:
Really sorry for the delay on this, my sincere apologies. Since this is a really small project and really simple implementation, forks are expected. I prefer avoid creating features to just right after breaking them.